doc/classes/NavigationServer2D.xml

```diff
@@ -736,7 +736,7 @@
 			<param index="0" name="obstacle" type="RID" />
 			<param index="1" name="layers" type="int" />
 			<description>
-				Set the obstacles's [code]avoidance_layers[/code] bitmask.
+				Sets the obstacle's [code]avoidance_layers[/code] bitmask.
```

Comment:
```suggestion
				Sets the obstacle's [code]avoidance_layers[/code] bitmask to [param layers].
```
Although, how weird to use code tags for this.
```suggestion
				Sets the given [param obstacle]'s avoidance layers to [param layers].
```
Would also apply to 3D I think.